### DISCUSSION TITLE
Return a different value in Rofi

### DISCUSSION THREAD

**Discussion by blueray453** [davatorium/rofi#1873] [Q&A] [answered]
Suppose I have a script like:

    #!/usr/bin/env bash

    echo -en "\tTags\tTitles\0nonselectable\x1ftrue\n"

    for item in ${items}; do
        title=$(title_of_item $item)
        tag=$(tag_of_item $item)
        icon=$(icon_of_item $item)
        echo -en "\t${tag}"
        echo -en "\t${title}"
        echo -en "\0icon\x1f${icon}\n"
    done

Here, I have two issues actually.

1. I want to return `$item` when a menu item is selected, given i am using `$item` to populate menu items but the value of `$item`  is not printed.  
2.  I am calling the script using `./script | rofi -dmenu -theme-str 'element-text { tab-stops: [ 1em,6em]; }'`. How to put this settings in the config using something like `\0theme-str....\x1f` so that I can call the script like  `./script | rofi -dmenu`

**Comment by DaveDavenport** [comment, marked as answer]
1. Return the index in ${items} (see rofi-dmenu manpage-> format option) then you can use the returned index to lookup item.
2. https://davatorium.github.io/rofi/1.7.5/rofi-script.5/#passing-mode-options   -> see the 'theme' option.
I am not 100% sure this will work for tabstop, but worth the try.

**Reply by blueray453** [reply]
`echo -en "\0theme\x1felement-text {  tab-stops: [ 1em,6em]; }"` works.
